src/Viz/VizProject.jl

```diff
@@ -0,0 +1,211 @@
+#=
+ MIT License
+
+ Copyright (c) 2010-present David A. Kopriva and other contributors: AUTHORS.md
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ --- End License
+=#
+
+const MODEL = 1; const GRID = 2; const MESH = 4; const EMPTY = 0
+const REFINEMENTS = 8; const ALL = 15
+
+
+"""
+    plotProject!(proj::Project, plotOptions::Int = 0)
+
+Plot objects specified by the `plotOptions`. Construct the `plotOptions` by the sum
+of what is to be drawn from the choices `MODEL`, `GRID`, `MESH`, `REFINEMENTS`.
+
+Example: To plot the model and the grid, `plotOptions = MODEL + GRID`. To plot
+just the mesh, `plotOptions = MESH`.
+
+To plot everything, `plotOptions = MODEL + GRID + MESH + REFINEMENTS`
+
+Contents are overlayed in the order: GRID, MESH, MODEL, REFINEMENTS
+"""
+function plotProject!(proj::Project, plotOptions::Int = 0)
```

Comment:
Optimally, this uses a plot recipe like we do for Trixi. But I recommend to create an issue for this and not block merging this PR